core/SConscript.unix

```diff
@@ -668,9 +668,13 @@ protobuf_blobs = env.Command(
 )
 env.Depends(protobuf_blobs, qstr_generated)
 
-RUST_PROFILE = 'dev' if ARGUMENTS.get('TREZOR_EMULATOR_DEBUGGABLE', 0) == '1' else 'release'
+if ARGUMENTS.get('TREZOR_EMULATOR_DEBUGGABLE', '0') == '1':
+    RUST_PROFILE = 'dev'
```

Comment:
Why not "debug"?

Reply:
With `debug` I'm getting:
```
$ make build_unix_debug
...
cd embed/rust; cargo build --profile debug --target-dir=../../build/unix/rust --no-default-features --features "model_tt"
error: profile name `debug` is reserved
To configure the default development profile, use the name `dev` as in [profile.dev]
```

According to https://doc.rust-lang.org/cargo/reference/profiles.html there's no default `debug` profile.

Reply:
Facepalm, sorry.